src/addons.code-workspace

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
You probably want to remove this file (from the current patch).

Reply:
heh, whoops